genders/nicky.json

```diff
@@ -0,0 +1,14 @@
+{
+    "names": ["Example"],
+    "type": "Gender name",
+    "presentation": {
+        "masc": 0.0,
+        "fem": 5.0,
+        "andro": 2.5,
+        "none": 2.5,
+        "custom": {}
+        },
+    },
+    "pronouns": ["she", "her", "hers"],
```

Comment:
This isn't json5
```suggestion
    "pronouns": ["she", "her", "hers"]
```

Reply:
thats what the example file has, so thats what im doing

Reply:
that's because there was stuff under it that you removed